addon/components/u-edit/shared-modules/drag-and-drop.js

```diff
@@ -0,0 +1,88 @@
+import Uploader from '@upfluence/ember-upf-utils/uploader';
+import Configuration from '@upfluence/ember-upf-utils/configuration';
+
+export default class ModuleBuilder {
+  constructor(session, toast) {
+    Object.assign(this, { session, toast });
```

Comment:
`this.session = session` (same for toast). let's keep the class usage "strict" even if this works because javascript ;)

Reply:
That's my bad, I had suggested to use this to @JulienVannier66  a while back because we had many arguments and it was more "clean" to assign them.

Reply:
So I change that to 
```javacript
this.session = session
this.toast = toast
```